src/index.html

```diff
@@ -1,12 +1,47 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="home_page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div 
+      class="
+      calendar
+      calendar__start-day-sun 
+      calendar__month-length-29"
```

Comment:
```suggestion
    <div 
      class="calendar calendar__start-day-sun calendar__month-length-29"
```
i think problr with linter just here